src/components/Article-comments.jsx

```diff
@@ -0,0 +1,81 @@
+import { React, useEffect } from "react";
+import { deleteComment } from "../utils/api";
+import { getComments } from "../utils/api";
+
+const ArticleComments = ({
+  open,
+  setComments,
+  comments,
+  commentChange,
+  setErr,
+  err,
+  setCommentChange,
+  username,
+  article_id,
+}) => {
+  useEffect(() => {
+    setCommentChange(false);
+    getComments(article_id)
+      .then((data) => {
+        function compare(a, b) {
+          const timeA = a.created_at;
+          const timeB = b.created_at;
+          let comparison = 0;
+          if (timeA > timeB) {
+            comparison = -1;
+          } else if (timeA < timeB) {
+            comparison = 1;
+          }
+          return comparison;
+        }
+        data.comments.sort(compare);
+        setComments(data.comments);
```

Comment:
Sorting is a very slow process - I would definitely recommend stepping into your BE at another time and implementing the sort functionality there so you can refactor this to not have to worry about it on the FE!